inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -849,6 +849,97 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
 
         return res;
     });
+
+    addSpecificCreator({"LRN_IE"}, [](const std::shared_ptr<::ngraph::Node>& node,
+                                      const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "Norm", details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<InferenceEngine::NormLayer>(attrs);
+        res->params = params;
+        return res;
+    });
+
+    addSpecificCreator({"MatMul"}, [](const std::shared_ptr<::ngraph::Node>& node,
+                                      const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "Gemm", details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<InferenceEngine::GemmLayer>(attrs);
+
+        auto transpUppercaseFirstChar = [](const std::string& transpose) -> std::string {
```

Comment:
Do we need this method? Why cannot we pass "true" and "false" instead of "True" and "False"? I thought that all should work.

Reply:
Method _on_adapter(..., ngraph::ValueAccessor<bool>&)_ used in visit_attributes() is casting read values from xml to lower case: https://github.com/openvinotoolkit/openvino/blob/6666e5fed3434637e538cf9d3219077c320c3c16/inference-engine/src/readers/ir_reader/ie_ir_parser.hpp#L194
On the other hand legacy representation expects not transformed parameter names (True/False). Functional test is complaining with following error: _ERROR: Layer fc in new network differs from reference parameter transpose_a (new, old): true, True_

Reply:
@sdurawa I discussed this problem with other guys. Let's always use true/false.

Could you remove this WA and change the test?

Reply:
WA is removed, test is corrected accordingly.